cis/libs/api.py

```diff
@@ -0,0 +1,53 @@
+import http.client
+import json
+import urllib
+
+
+class Person(object):
+    """Retrieve data from person api as needed.  Will eventually replace Mozillians API"""
+    def __init__(self, person_api_config):
+        """
+        :param person_api_config, a dictionary of configuration information about how to connect to person_api
+        """
+
+        # Audience is either https://person-api.sso.mozilla.com or https://person-api.sso.allizom.org
+        self.audience = person_api_config.get('audience')
```

Comment:
The audience is part of the OIDC API on the other side.  They don't necessarily have to be the same as the URL but in this case they are.  Though it may not always be that way for all endpoints.